### ISSUE TITLE
Integrity check fails: extra file calendar/appinfo/signature.json

### ISSUE THREAD

**Issue by keunes** [nextcloud/calendar#5095]
### Steps to reproduce

2. open https://cloud.site.me/settings/admin/overview

### Expected behavior

No warning is shown

### Actual behaviour

The following warning is shown:
Sommige bestanden hebben de integriteitscontrole niet doorstaan. Meer informatie over het oplossen van dit probleem is te vinden in de[ documentatie ↗ ](https://docs.nextcloud.com/server/26/go.php?to=admin-code-integrity). ([ Lijst met ongeldige bestanden… ](https://cloud.site.me/settings/integrity/failed)/[ Opnieuw scannen… ](https://cloud.site.me/settings/integrity/rescan?requesttoken=xxx))

The list with invalid files contains the following content:
```
Results
=======
- calendar
	- EXTRA_FILE
		- calendar/appinfo/signature.json

Raw output
==========
Array
(
    [calendar] => Array
        (
            [EXTRA_FILE] => Array
                (
                    [calendar/appinfo/signature.json] => Array
                        (
                            [expected] => 
                            [current] => 0d220d46747342a98daa3b5d821cde9e6e71cc2ab77768fc5afe6ec519d60879f8e95e78df135596cff48bc072956b232860f4c2980bbb1b93c6ae5dd1017b82
                        )

                )

        )

)
```

Is this a bug in the app, or something in my set-up (docker-based)?
Should/Can I delete this file?

### Calendar app version

4.3.1

### Server operating system

Raspberry Pi OS

### Web server

None

### Database engine version

None

### PHP engine version

None

### Nextcloud version

26.0.0

### Updated from an older installed version or fresh install

Updated from an older version

**Comment by ChristophWurst** [comment]
you probably update the calendar app manually, right? then delete the extraneous file.

when updating apps manually always remove the old directory. don't merge the new one into the old.

**Comment by keunes** [comment]
Thanks for the quick reply @ChristophWurst.

> you probably update the calendar app manually, right?

Actually I didn't. That is: I am running a [docker-based installation](https://github.com/nextcloud/docker) (not AIO), was on the default tag (that's `latest` I believe) and updated from Nextcloud 25.x to 26.0 by pulling, taking down & restarting containers. Apps were updated in the process.

> then delete the extraneous file.

I understand from this that I can safely delete the file. However, going into the Docker volume, I could find an 'apps' directory but 'calendar' is not in it (seemed to be mostly 'default' apps like 'comments', 'dashboard' and 'files'). So I'm not entirely sure where that file actually is.

**Comment by ChristophWurst** [comment]
Check your config.php. You probably have more than one app directories in ``apps_paths``. Then check if you have more than one copy of the app in those directories.

**Comment by keunes** [comment]
I checked my docker compose file and discovered that right in that `nextcloud-docker` folder I also have an `apps` folder (in addition to the one in the nextcloud data volume):
```
    volumes:
      - nextcloud:/var/www/html
      - ./config:/var/www/html/config
      - ./apps:/var/www/html/custom_apps
      - /nextcloud-data:/var/www/html/data
```

And the calendar app indeed has it's own corner in there. I guess these are the apps that I installed manually, while the ones in the `data` folder are the apps that are part of core (?)

In any case, removing the file `signature.json` did the trick. Thanks!
